pkgs/tools/networking/openssh/default.nix

```diff
@@ -46,6 +46,7 @@ stdenv.mkDerivation rec {
   # I set --disable-strip because later we strip anyway. And it fails to strip
   # properly when cross building.
   configureFlags = [
+    "--sbindir=\${out}/bin"
```

Comment:
The fixup script does work here, but I'd rather use the configure flag to point sbin at the bin dir in the first place instead of putting it in sbin and using the script to fix it up. Essentially, I consider the fixup script a bit of a hack, and I'd rather see it get less usage instead of more, considering there is a sanctioned way to do this via configure.

This should also be (very slightly) faster.